src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function (knex) {
+  return knex.schema.table('events_users', function (table) {
+    table.dropColumn('comment');
+    table.dropColumn('status');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('events_users');
```

Comment:
You would need to create columns `comment` and `status` here.

Reply:
Those columns are in the old version and are supposed to be dropped according to the recent db_diagram.
So why I should create them?

Reply:
Noted and feedback implemented!